stable/spring-cloud-data-flow/values.yaml

```diff
@@ -64,4 +74,16 @@ kafka:
     replicaCount: 1
 
 mysql:
+  enabled: true
   mysqlDatabase: dataflow
+
+## If NOT using the MySQL in this chart (mysql.enabled=false),
```

Comment:
this should probably be generic, something like "## If you are using an external database,"